lib/scarpe/html.rb

```diff
@@ -1,6 +1,8 @@
 module Scarpe
   class HTML
-    TAGS = %i[div p button ul li]
+    CONTENT_TAGS = %i[div p button ul li].freeze
+    VOID_TAGS = %i[input].freeze
```

Comment:
Freebie for someone else; if you were rewriting image, you could put it in here too etc. etc.